library/core/src/lib.rs

```diff
@@ -163,6 +163,7 @@
 #![feature(const_slice_index)]
 #![feature(const_is_char_boundary)]
 #![feature(const_cstr_methods)]
+#![feature(cmp_scalar)]
```

Comment:
nit: range_cmp_scalar is probably better (there's some benefit to being able to grep for these on stuff like cs.github.com or grep.app)

Reply:
Changed it!